apps/mobile/src/components/ContextSwitchMenu/ContextSwitchMenu.js

```diff
@@ -42,167 +71,109 @@ export default function ContextSwitchMenu ({ isExpanded, setIsExpanded }) {
     if (homePath) openURL(homePath)
   }
 
-  const handleLongPress = () => {
-    setIsPressing(true)
-    setIsExpanded(true)
-  }
-
-  const handlePressOut = () => {
-    setIsPressing(false)
-  }
-
   return (
-    <Animated.View className='flex-col h-full bg-theme-background z-50 w-full py-2 px-3'>
+    <View
+      className='h-full bg-theme-background z-50'
+      style={{ paddingTop: insets.top, paddingBottom: insets.bottom }}
+    >
       <FlatList
-        className='w-full'
         data={myGroups}
         keyExtractor={(item) => item.id}
         renderItem={({ item }) => (
           <ContextRow
-            onPress={handleOnPress}
-            item={item}
-            currentGroupSlug={currentGroup?.slug}
-            setIsExpanded={setIsExpanded}
             isExpanded={isExpanded}
-            onLongPress={handleLongPress}
+            item={item}
+            highlight={item?.slug === currentGroup?.slug}
+            onPress={handleOnPress}
+            onPressIn={handlePressIn}
             onPressOut={handlePressOut}
           />
         )}
         showsVerticalScrollIndicator={false}
+        onScrollBeginDrag={handleScrollStart}
+        onScrollEndDrag={handleScrollStop}
+        onMomentumScrollEnd={handleScrollStop}
+        scrollEventThrottle={16}
       />
-      <View className='w-full mt-auto bg-theme-background pt-4'>
-        <TouchableOpacity
-          onPress={() => openURL('/create')}
-          activeOpacity={0.7}
-          onLongPress={handleLongPress}
-          onPressOut={handlePressOut}
-          delayLongPress={200}
-          className='flex flex-row items-center gap-3 justify-start p-1'
-        >
-          <View
-            className='bg-primary relative flex flex-col items-center justify-center w-14 h-14 min-h-10 rounded-lg drop-shadow-md opacity-60 scale-90'
-          >
-            <Plus />
-          </View>
-          {isExpanded && (
-            <Text className='text-xl font-medium text-foreground flex-1 px-2 py-1 bg-foreground/20 rounded-md'>Create</Text>
-          )}
-        </TouchableOpacity>
-        <TouchableOpacity
-          onPress={() => Intercom.present()}
-          activeOpacity={0.7}
-          onLongPress={handleLongPress}
-          onPressOut={handlePressOut}
-          delayLongPress={200}
-          className='flex flex-row items-center gap-3 justify-start p-1'
-        >
-          <View className='bg-primary relative flex flex-col items-center justify-center w-14 h-14 min-h-10 rounded-lg drop-shadow-md opacity-60 scale-90'>
-            <CircleHelp />
-          </View>
-          {isExpanded && (
-            <Text className='text-xl font-medium text-foreground flex-1 px-2 py-1 bg-foreground/20 rounded-md'>Support</Text>
-          )}
-        </TouchableOpacity>
-      </View>
-    </Animated.View>
+      <ContextRow
+        bottomItem
+        isExpanded={isExpanded}
```

Comment:
for fwiw, I think these two should stay the same, regardless of isExpanded

Reply:
Not following... `isExpanded` is currently used here to reveal or hide the title, and that behavior is the same for both these bottom items and the contexts.